custom_components/yamaha_ynca/input_helpers.py

```diff
@@ -54,6 +54,8 @@ class Mapping:
     Mapping(ynca.Input.HDMI5, []),
     Mapping(ynca.Input.HDMI6, []),
     Mapping(ynca.Input.HDMI7, []),
+    Mapping(ynca.Input.OPTICAL1, []),
+    Mapping(ynca.Input.OPTICAL2, []),
```

Comment:
Could you move these to below MULTICH so the list stays alphabetical?

Reply:
Move it below MULTICH and switched USB and VAUX, so the list is truly alphabetical.